arcgis-java-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/skins/UtilityNetworkTraceSkin.java

```diff
@@ -0,0 +1,539 @@
+  public SimpleBooleanProperty isTraceInProgressProperty = new SimpleBooleanProperty(false);
+  public SimpleListProperty<UtilityNetwork> utilityNetworksProperty = new SimpleListProperty<>(FXCollections.observableArrayList());
+  public SimpleListProperty<UtilityNamedTraceConfiguration> traceConfigurationsProperty = new SimpleListProperty<>(FXCollections.observableArrayList());
+  public SimpleListProperty<UtilityNetworkTraceStartingPoint> startingPointsProperty = new SimpleListProperty<>(FXCollections.observableArrayList());
+  public SimpleListProperty<UtilityNetworkTraceOperationResult> traceResultsProperty = new SimpleListProperty<>(FXCollections.observableArrayList());
+  public SimpleObjectProperty<UtilityNetwork> selectedUtilityNetworkProperty = new SimpleObjectProperty<>();
+  public SimpleObjectProperty<UtilityNamedTraceConfiguration> selectedTraceConfigurationProperty = new SimpleObjectProperty<>();
+  public SimpleStringProperty traceNameProperty = new SimpleStringProperty();
+
+  @FXML ProgressIndicator utilityNetworkLoadingProgressIndicator;
+  // displays if no utility networks are found
+  @FXML VBox utilityNetworksNotFoundVBox;
+
+  @FXML TabPane tabPane;
+  // tab containing configuration options for running a trace
+  @FXML Tab newTraceTab;
+  @FXML BorderPane newTraceBorderPane;
+  // utility networks
+  @FXML VBox utilityNetworkSelectionVBox;
+  @FXML ComboBox<UtilityNetwork> utilityNetworkSelectionComboBox;
+  // named trace configuration
+  @FXML VBox traceConfigsNotFoundVBox;
+  @FXML ComboBox<UtilityNamedTraceConfiguration> traceConfigComboBox;
+  @FXML VBox traceConfigVBox;
+  // starting points
+  @FXML VBox startingPointsVBox;
+  @FXML ListView<UtilityNetworkTraceStartingPoint> startingPointsListView;
+  @FXML Label startingPointsPlaceholder;
+  @FXML Button addStartingPointButton;
+  @FXML Button cancelAddStartingPointsButton;
+  @FXML Button clearStartingPointsButton;
+  @FXML VBox addStartingPointProgressVBox;
+  @FXML Button cancelIdentifyStartingPointsButton;
+  @FXML TextField traceNameTextField;
+  // warnings
+  @FXML HBox aboveMinStartingPointsWarningHBox;
+  @FXML HBox insufficientStartingPointsWarningHBox;
+  // run trace
+  @FXML Button runTraceButton;
+  @FXML VBox traceInProgressVBox;
+  @FXML ProgressBar traceInProgressBar;
+  @FXML Label traceInProgressLabel;
+  @FXML Button cancelTraceInProgressButton;
+  // results
+  @FXML VBox noResultsFoundVBox;
+  @FXML Tab resultsTab;
+  @FXML VBox resultsVBox;
+  @FXML TabPane resultsTabPane;
+  @FXML Button clearResultsButton;
+
+  /**
+   * Constructor for all SkinBase instances.
+   *
+   * @param control The control for which this Skin should attach to.
+   */
+  public UtilityNetworkTraceSkin(UtilityNetworkTraceTool control) {
+    super(control);
+    // configure mapview related settings
+    controlMapView = skinnable.getMapView();
+    callout = controlMapView.getCallout();
```

Comment:
Hmm yep not ideal. Exploring this further. Will check a couple of options with the team and report back.

Reply:
Thanks for catching this, not great. I shared with the team and checked no one else was in the same boat. Since there are various scenarios where the control overrides other callout settings and vice versa, we agreed I should just remove this to avoid weird bugs, as it wasn't hugely essential functionality - more assistive in identifying starting points between the list and map which I think is still doable using the symbol swatch. We can see if a future callout implementation could be used here at some point.